src/LoRa.cpp

```diff
@@ -365,6 +385,23 @@ void LoRaClass::setSpreadingFactor(int sf)
   writeRegister(REG_MODEM_CONFIG_2, (readRegister(REG_MODEM_CONFIG_2) & 0x0f) | ((sf << 4) & 0xf0));
 }
 
+long LoRaClass::getSignalBandwidth()
```

Comment:
I'd prefer to leave this API out for now, unless we decide to have getters for all other parameters.

Reply:
Ok fair enough. Getters for the other settings could be good, but something for a separate pull.